codice/ddf
# DDF-5178 Migrated map drawing library to DDF.

#### What does this PR do?
Moves map drawing library to DDF.
#### Who is reviewing it? 
@jrnorth @samuelechu @bakejeyner 
#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.
@AzGoalie
@djblue

#### How should this be tested?
Run `yarn storybook` from `ui/packages/catalog-ui-search`
And inspect all of the components in the `map-drawing` folder.
Verify that all of the controls work.
Run `yarn test` from `ui/packages/catalog-ui-search`

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
